layouts/partials/footer.html

```diff
@@ -40,12 +40,15 @@
         {{ end }}
 
 
-        {{with ($.Scratch.Get "prevPage")}}
-            <a class="nav nav-prev" href="{{.URL}}" title="{{.Title}}"> <i class="fa fa-chevron-left"></i></a>
-        {{end}}
-        {{with ($.Scratch.Get "nextPage")}}
-            <a class="nav nav-next" href="{{.URL}}" title="{{.Title}}" style="margin-right: 0px;"><i class="fa fa-chevron-right"></i></a>
-        {{end}}
+	 {{$showPrevNext := (and (not .Params.disableNextPrev) (not .Site.Params.disableNextPrev))}}
```

Comment:
I think this is equivalent to `(or .Params.disableNextPrev .Site.Params.disableNextPrev)`. But I'm not sure this should be the condition to use.

Actually, I think page params should always override behaviour of site params when set. For example, we have a site and we choose to disable all nextprev for the site in the `config.toml`, but we want to enable it on a single page tough. It's not possible with this implementation.